`make simpletest` fails on tests/io/tri

with icestorm from master, yosys from master and arachne-pnr from master

<details>
 <summary>Build log</summary>

```
+ ../../bin/arachne-pnr tri.blif -o tri.txt
seed: 1
device: 1k
read_chipdb +/share/arachne-pnr/chipdb-1k.bin...
  supported packages: cb121, cb132, cb81, cm121, cm36, cm49, cm81, qn84, swg16tr, tq144, vq100
read_blif tri.blif...
prune...
instantiate_io...
pack...

After packing:
IOs          6 / 96
GBs          0 / 8
  GB_IOs     0 / 8
LCs          2 / 1280
  DFF        0
  CARRY      0
  CARRY, DFF 0
  DFF PASS   0
  CARRY PASS 0
BRAMs        0 / 16
WARMBOOTs    0 / 1
PLLs         0 / 1

place_constraints...
promote_globals...
  promoted 0 nets
  0 globals
realize_constants...
place...
  initial wire length = 133
  at iteration #50: temp = 6.39439, wire length = 68
  at iteration #100: temp = 2.67362, wire length = 17
  at iteration #150: temp = 0.022659, wire length = 9
  final wire length = 9

After placement:
PIOs       4 / 96
PLBs       2 / 160
BRAMs      0 / 16

  place time 0.01s
route...
  pass 1, 0 shared.

After routing:
span_4     0 / 6944
span_12    0 / 1440

  route time 0.00s
write_txt tri.txt...
+ icebox_vlog -s tri.txt
+ '[' -e tri.v ']'
+ diff -u tri.v out-tri.v
--- tri.v	2018-06-19 20:44:09.419402495 +0300
+++ out-tri.v	2018-06-19 20:49:53.384751024 +0300
@@ -6,8 +6,8 @@
 // IO Cell (0, 9, 1)
 assign io_0_9_1 = n4 ? n2 : 1'bz;
 
-assign n8 = /* LUT    1  9  3 */ 1'b0 ? 1'b0 : io_0_10_1 ? io_0_8_0 ? 1'b1 : 1'b0 : io_0_9_0 ? 1'b1 : 1'b0;
-assign n9 = /* LUT    1 10  5 */ 1'b0 ? 1'b0 : 1'b0 ? 1'b0 : io_0_10_1 ? io_0_10_0 ? 1'b1 : 1'b0 : 1'b1;
+assign n8 = /* LUT    1  9  3 */ (io_0_10_1 ? io_0_8_0 : io_0_9_0);
+assign n9 = /* LUT    1 10  5 */ (io_0_10_1 ? io_0_10_0 : 1'b1);
 /* FF  1  9  3 */ assign n2 = n8;
 /* FF  1 10  5 */ assign n4 = n9;
 
make: *** [Makefile:109: test] Error 1
```
</details>

--- a/tests/io/tri.v
+++ b/tests/io/tri.v
@@ -6,8 +6,8 @@ wire io_0_8_0, n2, io_0_9_0, n4, io_0_9_1, io_0_10_0, io_0_10_1, n8, n9;
 // IO Cell (0, 9, 1)
 assign io_0_9_1 = n4 ? n2 : 1'bz;
 
-assign n8 = /* LUT    1  9  3 */ 1'b0 ? 1'b0 : io_0_10_1 ? io_0_8_0 ? 1'b1 : 1'b0 : io_0_9_0 ? 1'b1 : 1'b0;
-assign n9 = /* LUT    1 10  5 */ 1'b0 ? 1'b0 : 1'b0 ? 1'b0 : io_0_10_1 ? io_0_10_0 ? 1'b1 : 1'b0 : 1'b1;
+assign n8 = /* LUT    1  9  3 */ (io_0_10_1 ? io_0_8_0 : io_0_9_0);
+assign n9 = /* LUT    1 10  5 */ (io_0_10_1 ? io_0_10_0 : 1'b1);
 /* FF  1  9  3 */ assign n2 = n8;
 /* FF  1 10  5 */ assign n4 = n9;
 

